app/models/ticket.js

```diff
@@ -21,8 +21,8 @@ export default ModelBase.extend({
   salesEndsAt          : attr('moment', { defaultValue: () => moment().add(10, 'days').startOf('day') }),
   minOrder             : attr('number', { defaultValue: 1 }),
   maxOrder             : attr('number', { defaultValue: 10 }),
-  minPrice             : attr('number'),
-  maxPrice             : attr('number'),
+  minPrice             : attr('number', { defaultValue: 1 }),
+  maxPrice             : attr('number', { defaultValue: 10 }),
```

Comment:
don't give a default 10 for maxPrice, let user specify it. 10 might not make sense in certain currencies.